resources/assets/js/store/index.js

```diff
@@ -57,6 +57,38 @@ let store = new Vuex.Store({
 
 	mutations: {
 
+		/**
```

Comment:
These mutations update the data model held in the state.

They update the data both in the currently edited page (state.page.pageData) IF that is the page whose data is being updated AND the data held in state.page.pages (the site structure).